charts/premium/traefik/values.yaml

```diff
@@ -499,3 +499,10 @@ portal:
   open:
     enabled: true
     path: /dashboard/
+ingress:
+  main:
+    integrations:
+      homepage:
+        href: https://{{  ( index .Values.ingress.main.hosts 0 ).host  }}/dashboard/
+        widget:
+          type: traefik
```

Comment:
why is this specified here?
Thats all automated afaik.

Reply:
I thought it would be needed. 
But just tested it and yes its not needed.